src/PowerState.cpp

```diff
@@ -52,11 +53,15 @@ uint8_t PowerState::enter_SD_card_write_state(uint8_t number){
   if(number == 1){
     Serial.println("entering SD 1 state");
     digitalWrite(SD_CARD_1_POWER_BUS, HIGH);
-    digitalWrite(SD_CARD_1_SS, LOW);
+    digitalWrite(SD_CARD_2_POWER_BUS, LOW);
+    digitalWrite(SD_CARD_1_SS, LOW); // on
```

Comment:
This looks weird but it's needed for the 2 glitchy SD card readers we have. It can be fixed when the hardware gets fixed.